### ISSUE TITLE
Dpkg can't install

### ISSUE THREAD

**Issue by richu99** [Eskender-B/cross-compile-opencv-4Termux#1]
I tried to install opencv_3.4.0_aarch64.deb
But I got error

dpkg: error processing archive opencv_3.4.0_aarch64.deb (--install):
 parsing file '/data/data/com.termux/files/usr/var/lib/dpkg/tmp.ci/control' near line 6 package 'opencv':
 end of file before value of field 'Homepage' (missing final newline)
Errors were encountered while processing:
 opencv_3.4.0_aarch64.deb

**Comment by eskender-f** [comment]
have you tried the following?

```dpkg -x /path/to/deb/file $PREFIX```

but first make sure you phone architecture is aarch64.

**Comment by richu99** [comment]
When i tried this command it does not show any installation

**Comment by eskender-f** [comment]
yes it directly extracts and does a copy paste (it is not a recommended way but it works)

to test if it really works do the following
```$ export LD_PRELOAD=$PREFIX/lib/libpython3.6m.so```

then inside python
```>>> import cv2```

if there is no error then it works.

**Comment by richu99** [comment]
→ dpkg -x opencv_3.4.0_aarch64.deb $PREFIX                              
→ dpkg -x opencv-dev_3.4.0_aarch64.deb $PREFIX
→ export LD_PRELOAD=$PREFIX/lib/libpython3.6m.so
→ python
Python 3.6.5 (default, Jun  8 2018, 01:39:02)
[GCC 4.2.1 Compatible Android (4691093 based on r316199) Clang 6.0.2 (https://a on linux
Type "help", "copyright", "credits" or "license" for more information.

>>> import cv2
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ModuleNotFoundError: No module named 'cv2'

 It also get error

**Comment by eskender-f** [comment]
what is your phone architecture?
```$ uname -a```
copy and paste the result for me to see.

**Comment by richu99** [comment]
Linux localhost 4.4.78-perf+ #1 SMP PREEMPT Mon Jun 11 01:37:40 CST 2018 aarch64 Android

**Comment by eskender-f** [comment]
the architecture is aarch64 (it should have worked)
I don't know man. It is still working for me. May be it has to do with the version of termux I am using which is an older one...

**Comment by fahadali32** [comment]
Is it work in arm71
